scripts/wrappers/dashboard_proxy.py

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/python3
 from subprocess import check_output
+import click
 
 
+@click.command(context_settings=dict(help_option_names=["-h", "--help"]))
 def dashboard_proxy():
+    """
+    Enables the dashboard add-on and configures port-forwarding
```

Comment:
nit; i'd prefer imperative for consistency.

```suggestion
    Enable the dashboard add-on and configures port-forwarding
```